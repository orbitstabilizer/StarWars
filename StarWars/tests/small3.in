5
Sector1 REPUBLIC
Sector2 REPUBLIC
Sector3 REPUBLIC
Sector4 REPUBLIC
Sector5 REPUBLIC
22
Sith sith1 10 10 10
Sith sith2 171 10 10
Sith sith3 10 10 100
Sith sith4 10 10 200
Sith sith5 500 10 10
Jedi jedi1 0 10 10
Jedi jedi2 0 10 10
Jedi jedi3 0 10 10
Jedi jedi4 0 10 10
Jedi jedi5 0 10 10
Officer officer1 COMMAND 1
Officer officer2 COMMAND 1
Officer officer3 GUNNERY 1
Officer officer4 GUNNERY 1
Officer officer5 PILOTING 1
Officer officer6 PILOTING 1
Officer officer7 ENGINEERING 1
Officer officer8 ENGINEERING 1
Officer officer9 TACTICAL 1
Officer officer10 TACTICAL 1
Sith sith6 10 10 10
Jedi jedi6 20 10 10
12
SeparatistDestroyer Destroyer1 1 1
2 1 11
SeparatistDestroyer Destroyer2 2 1
2 2 12
SeparatistDestroyer Destroyer3 3 1
2 3 13
SeparatistDestroyer Destroyer4 4 1
2 4 14
SeparatistDestroyer Destroyer5 5 1
2 5 15
RepublicCruiser Cruiser1 1 2
2 6 16
RepublicCruiser Cruiser2 2 2
2 7 17
RepublicCruiser Cruiser3 3 2
2 8 18
RepublicCruiser Cruiser4 4 2
2 9 19
RepublicCruiser Cruiser5 5 2
2 10 20
SeparatistDestroyer Destroyer6 5 10
1 21
RepublicCruiser Cruiser6 5 15
1 22
8
51 9 Armament 1000
10 6 1
10 7 2
10 8 3
10 9 4
10 5 10
10 12 11
10 6 5
