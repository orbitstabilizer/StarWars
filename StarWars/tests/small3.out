Warship Destroyer5 in (Sector5, 2)
sith5 1390
Warship Cruiser6 in (Sector5, 10)
jedi6 813
Warship Destroyer4 in (Sector4, 1)
sith4 786
Warship Destroyer2 in (Sector2, 1)
sith2 728
Warship Destroyer3 in (Sector3, 1)
sith3 586
Warship Destroyer1 is destroyed by Cruiser1 in (Sector1,1)
Warship Cruiser1 is destroyed by Destroyer5 in (Sector5,2)
Warship Cruiser2 is destroyed by Destroyer2 in (Sector2,1)
Warship Cruiser3 is destroyed by Destroyer3 in (Sector3,1)
Warship Cruiser4 is destroyed by Destroyer4 in (Sector4,1)
Warship Cruiser5 is destroyed by Destroyer5 in (Sector5,2)
Warship Destroyer6 is destroyed by Cruiser6 in (Sector5,10)
Sith sith5 is in Destroyer5
553
Sith sith4 is in Destroyer4
251
Sith sith2 is in Destroyer2
222
Sith sith3 is in Destroyer3
151
Jedi jedi6 is in Cruiser6
70
Sith sith1 is free
60
Sith sith6 is killed by jedi6
60
Jedi jedi1 is killed by sith5
40
Jedi jedi2 is killed by sith2
40
Jedi jedi5 is killed by sith5
40
Jedi jedi3 is killed by sith3
-50
Jedi jedi4 is killed by sith4
-59
Officer officer1 is killed by sith5
COMMAND 1
Officer officer2 is in Destroyer2
COMMAND 1
Officer officer3 is in Destroyer3
GUNNERY 1
Officer officer4 is in Destroyer4
GUNNERY 1
Officer officer5 is in Destroyer5
PILOTING 1
Officer officer6 is killed by sith5
PILOTING 1
Officer officer7 is killed by sith2
ENGINEERING 1
Officer officer8 is killed by sith3
ENGINEERING 1
Officer officer9 is killed by jedi4
TACTICAL 1
Officer officer10 is killed by sith5
TACTICAL 1
